lib/components/popover/popover.less

```diff
@@ -17,6 +17,9 @@
     --_po-arrow-after-r: unset;
     --_po-arrow-after-t: unset;
     --_po-arrow-after-bs: unset;
+    // content
+    --_po-topbar-height: var(--theme-topbar-height, calc(var(--su-static48) + var(--su-static8)));
```

Comment:
@dancormier Is it safe to assume that all pages will have a sticky topbar of these dimensions? I am genuinely asking because I don't have an opinion yet.

Reply:
Sounds good to me. A little comment about the assumption and the low impact if it is not respected could be added close to that variable in my opinion. I will leave it to you to decide. Either way consider this comment resolved. Thanks for taking the time to explain your reasoning.